pkg/capabilities/capabilities.go

```diff
@@ -76,10 +76,10 @@ type CapabilityRequest struct {
 }
 
 type TriggerEvent struct {
-	TriggerType    string
-	ID             string
-	Timestamp      string
-	BatchedPayload map[string]any
+	TriggerType string
+	ID          string
+	Timestamp   string
+	Payload     values.Value
```

Comment:
Nit/Unrelated: These could use comments. This whole file basically defines the Capability API but is sparsely commented - not ideal.

Reply:
This struct is documented in the design docs. I can add a comment saying that it's a trigger-specific payload but does that add any value?